domain/src/main/kotlin/no/nav/su/se/bakover/domain/beregning/Beregningsgrunnlag.kt

```diff
@@ -2,8 +2,22 @@ package no.nav.su.se.bakover.domain.beregning
 
 import no.nav.su.se.bakover.common.periode.Periode
 import no.nav.su.se.bakover.domain.beregning.fradrag.Fradrag
+import no.nav.su.se.bakover.domain.beregning.fradrag.FradragFactory
+import no.nav.su.se.bakover.domain.beregning.fradrag.FradragTilhører
+import no.nav.su.se.bakover.domain.beregning.fradrag.Fradragstype
 
 internal data class Beregningsgrunnlag(
     val periode: Periode,
-    val fradrag: List<Fradrag>
-)
+    private val forventetInntektPerÅr: Double,
+    private val fradragFraSaksbehandler: List<Fradrag>
+) {
+    val fradrag: List<Fradrag> = fradragFraSaksbehandler.plus(
+        FradragFactory.ny(
+            type = Fradragstype.ForventetInntekt,
+            beløp = forventetInntektPerÅr / 12.0,
```

Comment:
Burde beløp her da het månedsbeløp? Eller kunne vi bakt det inn i fradragstypen på noe vis? Slik at en kunne legge til fradag av forskjellige perioder.

Reply:
I lys av at vi nå er over på en "modell" hvor det for alle fradragsbeløp nå er snakk om månedsbeløp, kunne vi sikkert gjort det. Implementasjonene av interfacet har dette bakt inn i seg allerede, hvor den ene kun godtar enkeltmåneder, mens den andre (i bruk her) tillater vilkårlige perioder. Fradrag for vilkårlige/forskjellige perioder støttes dermed allerede.

Reply:
Fikser renaming i ny pr.